pkg/apis/v1alpha1/describe.go

```diff
@@ -0,0 +1,29 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Describe defines how to describe resources.
+type Describe struct {
+	// Timeout for the operation. Overrides the global timeout set in the Configuration.
+	// +optional
+	Timeout *metav1.Duration `json:"timeout,omitempty"`
+
+	// Resource type.
+	Resource string `json:"resource"`
```

Comment:
could it be kind ?

Reply:
it could, we would need to translate from kind to resource though